docs/manual/first-steps.rst

```diff
@@ -3,4 +3,4 @@ First steps
 
 The best way to get started with *GLASS* is to follow the examples__.
```

Comment:
This is actually not working right now, the correct version is

```suggestion
The best way to get started with *GLASS* is to follow the :doc:`/examples`.
```

and then the external link removed.

Reply:
Good spot, have tested it